okio-testing-support/src/commonMain/kotlin/okio/AbstractFileSystemTest.kt

```diff
@@ -742,6 +745,9 @@ abstract class AbstractFileSystemTest(
 
   @Test
   fun atomicMoveSourceIsDirectoryAndTargetIsFile() {
+    // `java.io` on Windows doesn't allow file renaming if the target already exists.
+    if (windowsLimitations && fileSystem::class.simpleName == "JvmSystemFileSystem") return
```

Comment:
should we have a separate function for this combination of limitations? 

```
  if (isJvmFileSystemOnWindows()) return
```

Reply:
I'll follow up on this